apollo-router/src/services/mod.rs

```diff
@@ -41,3 +41,9 @@ impl AsRef<Request> for Arc<http_ext::Request<Request>> {
         self.body()
     }
 }
+
+// set the supported `@defer` specification version to https://github.com/graphql/graphql-spec/pull/742/commits/01d7b98f04810c9a9db4c0e53d3c4d54dbf10b82
+pub(crate) const MULTIPART_DEFER_SPEC_PARAMETER: &str = "deferSpec";
+pub(crate) const MULTIPART_DEFER_SPEC_VALUE: &str = "20220824";
```

Comment:
```suggestion
pub(crate) const MULTIPART_DEFER_SPEC_VALUE: &str = "20220822";
```

clients are sending `deferSpec=20220822` can we use that instead?

Reply:
@prasek The motivation for picking `20220824` as the value seems justified/warranted since, as noted above, we're pinning to a commit of the specification on August **24**th.  I think clients should make the change here even though I realize this is a bit of a moving target at the moment.  It should settle down soon.

Reply:
Personally, I feel like the clarified behavior in that defer spec commit referenced above is noteworthy and worth abiding by, but if we agree together that it's not then let's consider ensuring our implementation to abides by the specification on the 22nd.  If we're going to have a `deferSpec` that looks like `YYYYMMDD` then it should probably match the spec as of that date.

That said, maybe we should choose a `deferSpec` that isn't date-based and perhaps a bit more flexible — like `202208.001`.

Reply:
the changes to the defer specification from August 24th are used in https://github.com/apollographql/router/pull/1640 so that will be the version we use, because we need the clarification on the `if` argument default behaviour

Reply:
Cool.  We can communicate that to the client teams.  Thanks!

Reply:
sounds good 👍